### PR TITLE
Review of session on Stats concepts

### PR DESCRIPTION
**Pull request by jamesmbaazam**

### CODE REVIEW

File: sessions/R-Stan-and-statistical-concepts.qmd [hunk 1]
```diff
@@ -99,7 +100,7 @@ mod$print(line_numbers = TRUE)
 ### Take 5 minutes
 Familarise yourself with the model above.
 Do you understand all the lines?
-Which line(s) define the parameter prior distribution(s), which one(s) the likelihood, and which one(s) the data that has to be supplied to the model?
```
Comment (kathsherratt) [inline sessions/R-Stan-and-statistical-concepts.qmd:102]: Personally I prefer the original version to make it clear that they might be on separate lines.

**Review by kathsherratt — COMMENTED** [review]
This is neat, thanks James!

File: sessions/R-Stan-and-statistical-concepts.qmd (hunk 1 above)
Comment (jamesmbaazam) [reply sessions/R-Stan-and-statistical-concepts.qmd]: I've reverted this in cbacf6863a26846197b3a47e49036cd999a0edd8.

**Comment by jamesmbaazam** [comment]
@kathsherratt Please feel free to approve and merge.

**Review by kathsherratt — APPROVED** [review]

**Pull request merged**